src/go/rpk/pkg/cli/cluster/quotas/import.go

```diff
@@ -0,0 +1,304 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package quotas
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/kafka"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/twmb/franz-go/pkg/kadm"
+	"github.com/twmb/types"
+	"gopkg.in/yaml.v3"
+)
+
+// quotasDiff represents a delta in the quotas after importing a quota.
+type quotasDiff struct {
+	EntityStr string `json:"entity,omitempty" yaml:"entity,omitempty"`
+	QuotaType string `json:"quota-type,omitempty" yaml:"quota-type,omitempty"`
+	OldValue  string `json:"old-value,omitempty" yaml:"old-value,omitempty"`
+	NewValue  string `json:"new-value,omitempty" yaml:"new-value,omitempty"`
+}
+
+func importCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var (
+		from      string
+		noConfirm bool
+	)
+	cmd := &cobra.Command{
+		Use:   "import",
+		Short: "Import client quotas",
+		Long: `Import client quotas.
+
+Use this command to import client quotas in the format produced by 
+'rpk cluster quotas describe --format json/yaml'.
+
+The schema of the import string is the same schema from 
+'rpk cluster quotas describe --format help':
+
+{
+  quotas: []{
+    entity: []{
+      name: string
+      type: string
+    }
+    values: []{
+      key: string
+      values: string
+    }
+  }
+}
```

Comment:
Rather than the describe help output, it'd probably be beneficial to say,
"""
(yaml format specified below that matches the 'describe --format help' format)

<put yaml format here>
"""